composer/algorithms/low_precision_layernorm/low_precision_layernorm.py

```diff
@@ -75,11 +75,12 @@ def apply(self, event: Event, state: State, logger: Logger) -> Optional[int]:
 
 class LPLayerNorm(torch.nn.LayerNorm):
 
-    def __init__(self, normalized_shape, eps=1e-05, elementwise_affine=True, device=None, dtype=None):
+    def __init__(self, normalized_shape, eps=1e-05, elementwise_affine=True, bias=True, device=None, dtype=None):
```

Comment:
Also probably need to put bias at the end to keep arg order?

Reply:
I'm following torch order

Reply:
I think better to be consistent